shaders/src/light_punctual.fs

```diff
@@ -142,6 +142,11 @@ Light getLight(const uint index) {
         light.contactShadows = bool((shadowBits >> 1u) & 0x1u);
         light.shadowIndex = (shadowBits >> 2u) & 0xFu;
         light.shadowLayer = (shadowBits >> 6u) & 0xFu;
+    } else {
+        light.castsShadows = false;
+        light.contactShadows = false;
+        light.shadowIndex = 0;
+        light.shadowLayer = 0;
```

Comment:
I wouldn't do a if-else for this -- just initialize the variables before the if.

Reply:
Done.  Also changed `0` to `0u` to (hopefully) fix the build errors seen on on some platforms.

Reply:
gotta love glsl!